README.md

```diff
@@ -1,25 +1,121 @@
 # sdcli
 
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/metabronx/sdcli/CI?label=tests&style=flat-square)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/metabronx/sdcli/ci.yaml?label=tests&style=flat-square)
 
-A metabronx command line tool for automating important grunt work.
+A command-line utility for executing essential but laborious tasks.
 
-Current features include:
+**Usage**:
 
-- Inviting members to the `metabronx` GitHub organization and teams, individually or in bulk:
+```console
+$ sdcli [OPTIONS] COMMAND [ARGS]...
+```
 
-  ```bash
-  sdcli gh invite --help
-  ````
+**Options**:
 
-- Assign members in bulk to teams in the `metabronx` GitHub organization:
+* `--help`: Show this message and exit.
 
-  ```bash
-  sdcli gh assign-teams --help
-  ````
+**Commands**:
 
-## Use
+* `gh`: Does things with GitHub's v3 REST API.
```

Comment:
Maybe "Performs GitHub operations"?

Reply:
You dont jive with the `Does things with` pattern?

Reply:
True, I do like that better...